chapters/mitigations-and-defensive-strategies/defense-mechanisms/activities/03-rwslotmachine3/sol/Makefile

```
@@ -0,0 +1,22 @@
+CFLAGS = -m32 -fno-stack-protector -Wall -Wno-format-security -no-pie
```

```suggestion
CFLAGS = -m32 -fno-stack-protector -Wall -Wno-format-security
LDFLAGS = -no-pie
```

`-no-pie` is a linker flag.